lib/widgets/message_list.dart

```diff
@@ -896,11 +896,14 @@ class RecipientHeader extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    final isTopicNarrow = narrow is TopicNarrow;
+    final isDmNarrow = narrow is DmNarrow;
```

Comment:
No need for separate variables here, just say `narrow is TopicNarrow` or `narrow is DmNarrow` inline.